src/llm/backends/codex.py

```
@@ -0,0 +1,436 @@
+                    response = getattr(event, "response", None)
+                    finish_reason = self._finish_reason(response)
+                    output_tokens = self._usage_output_tokens(getattr(response, "usage", None))
+
+            if output_tokens is None:
+                try:
+                    final_response = await stream.get_final_response()
+                    output_tokens = self._usage_output_tokens(
+                        getattr(final_response, "usage", None)
+                    )
+                    finish_reason = finish_reason or self._finish_reason(final_response)
+                except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)